tests/Command/CheckAllSchemasAreValidAvroCommandTestTest.php

```diff
@@ -126,6 +228,29 @@ public function testOutputWhenAllValid():void
         self::assertEquals(0, $commandTester->getStatusCode());
     }
 
+    public function testOutputWithDefaultTypeParsingException():void
+    {
+        file_put_contents(
+            sprintf('%s/test.schema.default.avsc', self::SCHEMA_DIRECTORY),
+            self::DEFAULT_VALUE_SCHEMA
+        );
+
+        $application = new Application();
+        $application->add(new CheckAllSchemasAreValidAvroCommand());
+        $command = $application->find('kafka-schema-registry:check:valid:avro:all');
+        $commandTester = new CommandTester($command);
+
+        $commandTester->execute([
+            'schemaDirectory' => self::SCHEMA_DIRECTORY
+        ]);
+
+        $commandOutput = trim($commandTester->getDisplay());
+
+        self::assertStringContainsString('Following schemas are not valid Avro', $commandOutput);
+        self::assertStringContainsString('* test.schema.default', $commandOutput);
```

Comment:
maybe you can also check that `Default is not in type` is contained, since this is what we are expecting here, correct?

Reply:
Actually the same error as before is expected, that exception is only used to mark schema as failed using existing logic in `catch`.